consul/audit/index.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express')
+var app = express()
+
+app.get('/', function (req, res) {
+    res.send('Hello World!')
+})
+app.get('/health', function (req, res) {
+    res.send('I am healthy!')
```

Comment:
I'm not opposed to a node app, but will just note that most examples have been standardizing on the helloworld app in https://github.com/linkerd/linkerd-examples/tree/master/docker/helloworld

But I guess consul needs a health check? Don't think the golang helloworld app has that at the moment.